ui/apps/platform/cypress/helpers/visit.js

```diff
@@ -30,3 +30,26 @@ export function visit(pageUrl, requestConfig, staticResponseMap) {
     cy.wait(['@featureflags', '@mypermissions', '@config/public']);
     waitForResponses(requestConfig);
 }
+
+/*
+ * Visit page to test conditional rendering for user role permissions.
+ *
+ * { body: { resourceToAccess: { … } } }
+ * { fixture: 'fixtures/wherever/whatever.json' }
+ */
+export function visitWithPermissions(
+    pageUrl,
+    permissionsStaticResponse,
+    requestConfig,
+    staticResponseMap
+) {
+    cy.intercept('GET', api.featureFlags).as('featureflags');
+    cy.intercept('GET', api.roles.mypermissions, permissionsStaticResponse).as('mypermissions');
```

Comment:
This is worth a dedicated slot in UI meeting, because it seems to be a low-cost way to start testing views which differ by permissions.